neuromation/cli/root.py

```diff
@@ -187,3 +195,19 @@ def _process_chunk(self, chunk: bytes, printable: bool) -> List[str]:
             for line in self._process_chunk(chunk, context.response_printable)
         ]
         self._print_debug(lines)
+
+    def _sanitize_header_value(self, text: str) -> str:
+        for token in self._find_all_tokens(text):
+            token_safe = self._sanitize_token(token)
+            text = text.replace(token, token_safe)
+        return text
+
+    def _sanitize_token(self, token: str, tail_len: int = 5) -> str:
+        assert 0 < tail_len, "tail too short"
```

Comment:
The assertion is not appropriate here: I don't want to crash a program if the token is short.

Reply:
ok I agree these assertions shouldn't abort the execution, thank you.